app/services/pagarme/request/pagarme.rb

```diff
@@ -0,0 +1,51 @@
+class Pagarme::Request::Pagarme
```

Comment:
Maybe change this in a future PR to `class Pagarme::Batch::Request` or anything else, up to you.